### ISSUE TITLE
App receives push notifications but crashes when opened

### ISSUE THREAD

**Issue by Paoleddu** [arma7x/telekram#10]
After having sideloaded the app through WebIDE and having successfully logged in, the app crashed (I must also add that this happened the second time I reinstalled the app, the first time I managed to get into the settings before it crashed).
The apps still manages to show push notifications, not all of them but whenever it connects to the Internet I suppose, though if I open the notification the app freezes on the main pages (it doesn't display the profile pictures) and force exits. 
I'm using a Nokia 6300 4G on KaiOS 2.5.4 with only debugging active, no root.

**Comment by Paoleddu** [comment]
No, unfortunately the app crashes also when I open it normally

**Comment by arma7x** [comment]
You install the package from bhstore or build it directly from this depository ?

**Comment by arma7x** [comment]
Maybe this issue is related to KaiOS firmware, latest update make the quite slow and unresponsive. I ALSO don't have any issues running Telekram on my Nokia 8110 4G

**Comment by Paoleddu** [comment]
I don't know, it might be caused by KaiOS
I've noticed that a few times I could enter a chat, but when I pressed the "Message" button it crashed
Later this afternoon I'll shot a couple of videos of the problems

**Comment by Paoleddu** [comment]
https://user-images.githubusercontent.com/87008838/180418975-fe92b75f-c472-464a-8ed6-384159b59095.mp4

**Comment by arma7x** [comment]
This is an OOM(Out Of Memory) issue. How many chats (personal, bot and channel) on your home screen?

**Comment by Paoleddu** [comment]
Around 70

**Comment by Paoleddu** [comment]
Also, two of the group chats I'm in are each over 2000 members, that might be an issue I guess

**Comment by Paoleddu** [comment]
Ok, I've tried archiving some of them, comprised the big group chats, and now there are about 40 chats in the main page, but unfortunately it still crashes

**Comment by arma7x** [comment]
This Telegram client does not support Archived Chats

**Comment by Paoleddu** [comment]
Ok, so for my use case it's an unsolvable problem.
Thanks anyway for your patience

**Comment by arma7x** [comment]
Yes. But, i'm working on new Telegram client https://github.com/arma7x/svelte-telekram, once it's stable I will send you a zip archive for testing purpose

**Comment by Paoleddu** [comment]
With pleasure
I've moved to an Energizer E241S, it should work the same

**Comment by Paoleddu** [comment]
Well, on the Energizer E241s (firmware 2.5.1.2) telekram, although slowly, works without going OOM, it even opens from the notifications
I guess that the 6300 4G is really poorly optimized
